### PR TITLE
popup export script Bootstrap convertion

### PR DESCRIPTION
**Pull request by robertogagliotta**
Setup varios <head> elements
Header::setupHeader();

### CODE REVIEW

**Comment by bradymiller** [comment]
After we are done with this script, I'll give you a set of instructions to clean up your master branch (which will avoid bringing in all the merge commits in future Pull Requests).

**Comment by bradymiller** [comment]
The code looks good to me (I am new at bootstrap so it will be good to get @robertdown 's blessing on this). I'll test it out this evening.

**Comment by robertdown** [comment]
Will review tomorrow. @robertogagliotta can you clean up this PR a bit - it's showing 71 commits and lots of merges. Can help with that if you need, but it's basically just getting your local master branch up to date and then rebasing this branch

**Comment by robertdown** [comment]
Guess it helps to read all of the comments lol

**Comment by bradymiller** [comment]
yep, we can work on cleaner PR's for the future. It will be easy after I show how to clean up @robertogagliotta 's master.

File: custom/export_xml.php [hunk 1]
```diff
@@ -252,25 +255,36 @@ function addInsurance($row, $seq) {
...
+<?php Header::setupHeader(); ?>
+<title><?php echo xlt('Export Patient Demographics'); ?></title>
 </head>
 <body>
-
-<p><?php xl('The exported data appears in the text area below.  You can copy and
-paste this into an email or to any other desired destination.','e'); ?></p>
-
-<center>
 <form>
```
Comment (robertdown) [inline custom/export_xml.php]: No need for this. I know it was already in the prior to your work but we can pull it.

File: custom/export_xml.php [hunk 2]
```diff
@@ -252,25 +255,36 @@ function addInsurance($row, $seq) {
...
+           </div>
+        </div> 
+     </div>
+     <div class="row">
+        <div class="col-xs-12">
+           <div class="form-group">
+              <textarea name="export_data" class=form-control cols="50" rows="18"><?php echo text($out) ?></textarea>
+           </div>
+           <div class="form-group">
+              <div class="col-sm-offset-2 col-sm-10">
```
Comment (robertdown) [inline custom/export_xml.php]: Should be `col-xs-12 text-right`

File: custom/export_xml.php (hunk 1 above)
Comment (robertdown) [inline custom/export_xml.php]: I know this form tag was here before your changes, but we can go ahead and pull it, there's not need for it

File: custom/export_xml.php [hunk 3]
```diff
@@ -252,25 +255,36 @@ function addInsurance($row, $seq) {
...
+                 <div class="btn-group" role="group">
+                    <button type="button" class="btn btn-default btn-save" onclick="window.close()"><?php echo xlt("OK"); ?></button>
```
Comment (robertdown) [inline custom/export_xml.php]: I think better text would be "Close this tab" along with a negative-action icon (`btn-cancel` instead of `btn-save`)

**Comment by robertogagliotta** [comment]
This is the rebase/merge commit. I did find find other way to add it to the PR.
https://github.com/robertogagliotta/openemr/commit/3c57a945cdcb89c1e5c77bf9721bbf47d0489eb6

**Comment by robertogagliotta** [comment]
@robertdown 
ready for review. @bradymiller  The PR is messy, I need help to clean it up

**Comment by bradymiller** [comment]
@robertogagliotta ,
Just keep working on this branch and don't worry about how messy the commits and PR is. I can fix it when it's ready to come into the codebase. And then at that point, I'll give you a set of instructions to clean up your master branch which will then avoid the messy PR's in the future.
-brady

File: custom/export_xml.php [hunk 4]
```diff
@@ -252,25 +255,35 @@ function addInsurance($row, $seq) {
...
+<?php Header::setupHeader(); ?>
+<title><?php echo xlt('Export Patient Demographics'); ?></title>
 </head>
 <body>
-
-<p><?php xl('The exported data appears in the text area below.  You can copy and
-paste this into an email or to any other desired destination.','e'); ?></p>
-
-<center>
 <form>
```
Comment (bradymiller) [inline custom/export_xml.php]: delete form tag

File: custom/export_xml.php [hunk 5]
```diff
@@ -252,25 +255,35 @@ function addInsurance($row, $seq) {
...
+        </div>
+     </div>
+  </div> 
 
-<textarea rows='10' cols='50' style='width:95%' readonly>
-<?php echo $out ?>
-</textarea>
-
-<p><input type='button' value=<?php xl('OK','e','\'','\''); ?> onclick='window.close()' /></p>
 </form>
```
Comment (bradymiller) [inline custom/export_xml.php]: delete /form tag

File: custom/export_xml.php [hunk 6]
```diff
@@ -252,25 +255,35 @@ function addInsurance($row, $seq) {
...
-
-<center>
 <form>
+  <div class="container">
+     <div class="row">
+        <div class="col-xs-12">
+           <div class="page-header">
+           </div>
+        </div>  
+     </div>
```
Comment (bradymiller) [inline custom/export_xml.php]: delete 6 lines above. Since there is no longer a page-header, no need to include this code.

**Comment by bradymiller** [comment]
@robertogagliotta ,
Only a couple minor things left to do (see my comments). When those minor things are addressed, just let me know (then I'll fix up the branch and bring into codebase and then provide an instruction set to clean up your master branch).

**Comment by robertogagliotta** [comment]
https://github.com/robertogagliotta/openemr/commit/ed2aa9a14fc91714fc0c64e22f909eb13465a419#diff-59dfaf04499e616fb1f71e654a4610f5
ready for the Third review. @robertdown  @bradymiller

**Comment by bradymiller** [comment]
nice, I just brought this into the codebase:
https://github.com/openemr/openemr/commit/a77d6cc15090df6a0f33ba61c215ba32b3e48713

I also added the following commit after it with a couple changes:
https://github.com/openemr/openemr/commit/690b2cf8e88308c06880238859fcab5a17c66bdf
changes were:
1. body_top class added to body tag since this sets the background color from the css theme
2. empty form-group div class added add top to add some space above the textarea field
3. changed OK label to Close

**Comment by bradymiller** [comment]
Now it's time to clean your master branch :)
Follow these instruction to clean your master branch:
https://github.com/openemr/openemr/pull/657#issuecomment-295630669

**Comment by bradymiller** [comment]
After clean your master branch, then recommend converting the script at Popup->Import to bootstrap as next project.

**Comment by bradymiller** [comment]
@robertogagliotta ,
Beautiful master branch over there :)
https://github.com/robertogagliotta/openemr/commits/master

**Pull request closed without merging**